canopeum_frontend/src/main.tsx

```diff
@@ -26,8 +25,6 @@ if (root == null) {
   throw new Error('Could not find element #root to bootstrap React.')
 } else {
   createRoot(root).render(
-    <StrictMode>
-      <App />
-    </StrictMode>,
+    <App />,
```

Comment:
Why remove the StrictMode ? This change hides runtime issues during development

Reply:
Don't really remember the reason I deleted it, I put it back